tso-server/bin/omid-env.sh

```diff
@@ -22,6 +22,14 @@
 # Check if HADOOP_CONF_DIR and HBASE_CONF_DIR are set
 # ---------------------------------------------------------------------------------------------------------------------
 
-if [ -z ${HADOOP_CONF_DIR+x} ]; then echo "WARNING: HADOOP_CONF_DIR is unset"; else echo "HADOOP_CONF_DIR is set to '$HADOOP_CONF_DIR'"; fi
-if [ -z ${HBASE_CONF_DIR+x} ]; then echo "WARNING: HBASE_CONF_DIR is unset"; else echo "HBASE_CONF_DIR is set to '$HBASE_CONF_DIR'"; fi
+if [ -z ${HADOOP_CONF_DIR+x} ];
+  then echo "WARNING: HADOOP_CONF_DIR is unset";
+  HADOOP_CONF_DIR=/etc/hadoop/conf
```

Comment:
I am not sure about this.
These are the bigtop standard directories, but they are not used by HBase by default.

Reply:
At least check if there is an exepected config file in that directory before setting it.

Reply:
or at least that the direcory itself exists.